src/argilla/client/sdk/v1/datasets/api.py

```diff
@@ -215,6 +217,43 @@ def get_records(
     return handle_response_error(response)
 
 
+# TODO: Add small docstring to the function.
+# TODO: Add the rest of search supported parameters.
+def search_records(
+    client: httpx.Client,
+    id: UUID,
+    query: FeedbackRecordsSearchQuery,
+    # TODO: This should be named response_statuses (like we are already doing in the API) but get_records is using response_status
+    response_status: Optional[List[FeedbackResponseStatusFilter]] = None,
+    metadata_filters: Optional[List[str]] = None,
+    sort_by: Optional[List[str]] = None,
+    offset: int = 0,
+    limit: int = 50,
+) -> Response[Union[FeedbackRecordsSearchModel, ErrorMessage, HTTPValidationError]]:
+    url = f"/api/me/datasets/{id}/records/search"
```

Comment:
Once we have the "admin" endpoint, we should change this by `/api/v1/datasets/{id}/records/search`
```suggestion
    url = f"/api/v1/me/datasets/{id}/records/search"
```